model/proto/model.proto

```diff
@@ -134,6 +134,8 @@ message Span {
     (gogoproto.customname) = "ProcessID"
   ];
   repeated string warnings = 12;
+  //indicates whether this is the final span or an intermediate span
```

Comment:
`// indicates that the span has not been finished yet and only contains partial data`

Reply:
Yeah, sounds better ;)